tdrs-backend/tdpservice/users/urls.py

```diff
@@ -6,7 +6,7 @@
 
 router = DefaultRouter()
 
-router.register("users", views.UserViewSet)
+router.register("users/user", views.UserViewSet)
```

Comment:
this doesn't seem to break anything on the frontend, but it changes the url structure and i was curious why?
`/v1/users` being the list page and `/v1/users/{id}` being the details makes sense to me vs
`v1/users/user` being list and `/v1/users/user/{id}` details